python/taichi/math/mathimpl.py

```diff
@@ -68,6 +68,19 @@ def uvec4(*args):
     return ti.types.vector(4, _get_uint_ip())(*args)  # pylint: disable=E1101
 
 
+mat2 = ti.types.matrix(2, 2, impl.get_runtime().default_fp)  # pylint: disable=E1101
```

Comment:
Should `mat2, mat3, mat4` use the same way of definition to avoid the precision bug?

Reply:
Thanks! I didn't realize this doesn't solve the bug actually.